ciao-controller/openstack_volume.go

```diff
@@ -297,14 +297,17 @@ func (c *controller) ListVolumesDetail(tenant string) ([]block.VolumeDetail, err
 		vol.ID = data.ID
 		vol.Size = data.Size
 		vol.OSVolTenantAttr = data.TenantID
-		vol.CreatedAt = &data.CreateTime
+		createdAt := data.CreateTime
+		vol.CreatedAt = &createdAt
```

Comment:
Or better ... maybe I can iterate over the slice by index and take the addresses of the members in the original slice.